docs/content/latest/explore/ysql-language-features/gin.md

```diff
@@ -47,14 +47,12 @@ However, extension support is still in progress:
 
 ### CREATE INDEX
 
-Create the index using `USING ybgin` to specify the index access method:
+Create the index using `USING gin` to specify the index access method:
 
 ```sql
-CREATE INDEX ON mytable USING ybgin (mycol);
-```
+CREATE INDEX <optional name> ON <table> USING GIN (<column>);
 
-The `gin` access method is reserved for temporary relations while `ybgin` is for Yugabyte-backed relations.
-You can still specify `USING gin`, and, if `mytable` is not a temporary table, it will be automatically substituted for `ybgin`.
```

Comment:
This should still be called out in docs.  cc: @m-iancu

Reply:
Not required for Explore section